File: solutions/julia/darts/1/darts.jl
function score(x, y)
    if sqrt(x*x+y*y)<=1
        return 10
    elseif sqrt(x*x+y*y)<=5
        return 5
    elseif sqrt(x*x+y*y)<=10
        return 1
    end
    return 0
end

File: solutions/fortran/darts/1/darts.f90
module darts
  implicit none

contains

  function score(x, y) result(points)
    real, intent(in):: x, y
    real :: r
    integer :: points
    r = sqrt((x * x) + (y * y))
    !print *, x, y, r
    if (r > 10) then 
            points = 0   
    else if (r > 5) then
            points = 1
    else if (r > 1) then
            points = 5
    else
            points = 10
    end if
    !print *, points
  end function score

end module darts
